<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscUi</class>
 <widget class="QWidget" name="DiscUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DistDetails</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Device</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="devices_combo">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="volume_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="volume_area_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>415</width>
        <height>872</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{ 
	background-color : palette(base);
	color : palette(text);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Device Details</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>37</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="device_address_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="device_address_label">
           <property name="text">
            <string>Device Address</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="drive_type_label">
           <property name="text">
            <string>Drive Type</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="drive_type_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="firmware_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="firmware_label">
           <property name="text">
            <string>Drive Firmware Version</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="cdr_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="cdr_label">
           <property name="text">
            <string>CD Writer</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="cdrw_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="cdrw_label">
           <property name="text">
            <string>CD ReWritable</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="dvd_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="dvd_label">
           <property name="text">
            <string>DVD</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="dvdr_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="dvdr_label">
           <property name="text">
            <string>DVD Writer</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="dvdrw_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="dvdrw_label">
           <property name="text">
            <string>DVD ReWritable</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="dvdrdl_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="dvdrdl_label">
           <property name="text">
            <string>Double Layer DVD Writer</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="dvdram_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="dvdram_label">
           <property name="text">
            <string>DVD Ram</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="dvdplusr_label">
           <property name="text">
            <string>DVD+ Writer</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="dvdplusr_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="dvdplusrw_label">
           <property name="text">
            <string>DVD+ ReWritable</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="dvdplusrw_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="dvdplusrwdl_label">
           <property name="text">
            <string>Double Layer DVD+ ReWritable</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLabel" name="dvdplusrwdl_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="dvdplusrdl_label">
           <property name="text">
            <string>Double Layer DVD+ Writer</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLabel" name="dvdplusrdl_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="bd_label">
           <property name="text">
            <string>Blu-Ray</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLabel" name="bd_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="bdr_label">
           <property name="text">
            <string>Blu-Ray Writer</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLabel" name="bdr_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="hddvd_label">
           <property name="text">
            <string>HD-DVD</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QLabel" name="hddvd_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="hddvdr_label">
           <property name="text">
            <string>HD-DVD Writer</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QLabel" name="hddvdr_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <widget class="QLabel" name="hddvdrw_label">
           <property name="text">
            <string>HD-DVD ReWritable</string>
           </property>
          </widget>
         </item>
         <item row="18" column="1">
          <widget class="QLabel" name="hddvdrw_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Disc Details</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>37</number>
         </property>
         <property name="leftMargin">
          <number>25</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="disc_type_label">
           <property name="text">
            <string>Current Disc Type</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="disc_type_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="disc_blank_label">
           <property name="text">
            <string>Disc Is Blank</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="disc_blank_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="disc_capacity_label">
           <property name="text">
            <string>Disc Capacity</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="disc_capacity_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="read_speed_label">
           <property name="text">
            <string>Read Speed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="read_speed_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="removable_media_label">
           <property name="text">
            <string>Removable Media Available</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="removable_media_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Volume Details</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="horizontalSpacing">
          <number>37</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="volume_label">
           <property name="text">
            <string>Volume Label</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="fstype_label">
           <property name="text">
            <string>File System Type</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="fstype_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="fsversion_label">
           <property name="text">
            <string>File System Version</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="fsversion_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="disc_type_label_2">
           <property name="text">
            <string>Disc Type</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="disc_type_text_2">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="mountpoint_label">
           <property name="text">
            <string>Mount Point</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="mountpoint_text">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="appendable_label">
           <property name="text">
            <string>Appendable</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="appendable_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="volume_size_label">
           <property name="text">
            <string>Volume Size</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="volume_size_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="block_size_label">
           <property name="text">
            <string>Block Size</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="block_size_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="audio_label">
           <property name="text">
            <string>Audio</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="audio_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="vcd_label">
           <property name="text">
            <string>VCD</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="vcd_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="vbr_label">
           <property name="text">
            <string>Video BluRay</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLabel" name="vbr_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="vdvd_label">
           <property name="text">
            <string>Video DVD</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLabel" name="vdvd_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="data_label">
           <property name="text">
            <string>Data Disc</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="data_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="svcd_label">
           <property name="text">
            <string>Super VCD</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="svcd_text">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="volume_text">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
